app/src/main/java/com/example/kuba10/mypokemonplaces/ChooseFragment/PokemonImageAdapter.java

```diff
@@ -1,159 +0,0 @@
-package com.example.kuba10.mypokemonplaces.ChooseFragment;
-
-import android.content.Context;
-import android.graphics.Point;
-import android.support.v7.widget.RecyclerView;
-import android.view.Display;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.view.WindowManager;
-import android.widget.ImageView;
-import android.widget.ProgressBar;
-import android.widget.RelativeLayout;
-
-import com.example.kuba10.mypokemonplaces.FragmentListener;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-import com.squareup.picasso.Callback;
-import com.squareup.picasso.MemoryPolicy;
-import com.squareup.picasso.Picasso;
-
-
-import java.util.List;
-
-
-public class PokemonImageAdapter extends RecyclerView.Adapter<PokemonImageAdapter.ViewHolder> {
-
-
-    private Context context;
-    private List<PokemonGo> pokemonGo_data_list;
-    private FragmentListener fragmentListener;
-    ChooseFragment fragment;
-
-
-
-
-
-    public PokemonImageAdapter(Context context, FragmentListener fragmentListener, ChooseFragment fragment) {
-        this.context = context;
-        this.fragment = (ChooseFragment) fragment;
-        this.fragmentListener = fragmentListener;
-        pokemonGo_data_list = fragmentListener.getPokemonList();
-
-    }
-
-
-    @Override
-    public PokemonImageAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
-
-        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.choose_image_card, parent, false);
-
-
-
-
-        return new ViewHolder(view);
-
-    }
-
-    @Override
-    public void onBindViewHolder(final PokemonImageAdapter.ViewHolder holder, int position) {
-
-
-        ViewGroup.LayoutParams params = holder.imageContainer.getLayoutParams();
-
-        params.height = getScreenWidth() / 4;
-        params.width = getScreenWidth() / 4;
-        holder.imageContainer.setLayoutParams(params);
-
-
-        holder.loader.setVisibility(View.VISIBLE);
-        holder.imageView.setVisibility(View.VISIBLE);
-        final ProgressBar progressView = holder.loader;
-
-
-
-            Picasso.with(context)
-                    .load(pokemonGo_data_list.get(position).getImg())
-                    .memoryPolicy(MemoryPolicy.NO_CACHE, MemoryPolicy.NO_STORE)
```

Comment:
Umyślnie nie chcesz cachować? Jedną z głównych zalet libek do obrazków jest to, że cachują.

Reply:
No i popsułem picasso... A tak ładnie dziłałao.